marlowe-runtime/doc/tutorial.md

```diff
@@ -29,7 +29,6 @@ In a shell terminal, clone the Marlowe Cardano git repository and launch a Jupyt
 ```bash
 git clone https://github.com/input-output-hk/marlowe-cardano.git
 cd marlowe-cardano/marlowe-runtime/doc/
-nix run ../../marlowe-cli
```

Comment:
I don't understand what this is supposed to do.
The syntax is incorrect. It should be 
```
nix run ../../#marlowe-cli
``` 

However I don't see why it would be different from the setup on line 42 in the same file?

Reply:
I think this used to work because there was a nix flake in that folder. This change is okay for now, but I'll create a new ticket for us to thoroughly review this documentation.